pipeline-steps-explained.MD

```diff
@@ -0,0 +1,89 @@
+Dynatrace AppMon Pipeline steps
+-------------------------------
+Requirements: Jenkins 2.0+ with <a href="https://wiki.jenkins.io/display/JENKINS/Pipeline+Plugin">Pipeline Plugin</a>.
+
+Note: If you are using Jenkins 2.7.2 (or earlier), please ensure that your <a href="https://wiki.jenkins.io/display/JENKINS/Pipeline+Nodes+and+Processes+Plugin">Pipeline: Nodes and Processes plugin</a> version is not higher than 2.13.
+
+Note: For Maven projects, please use <a href="https://wiki.jenkins.io/display/JENKINS/Maven+Project+Plugin">Maven Integration Plugin</a> version 2.13 (or higher).
+
+Global configuration remains unchanged and is described [here](README.MD) in Global Settings section.
+
+---
+#### **appMonBuildEnvironment**
+Corresponds to 'Use Dynatrace AppMon to monitor tests' from 'Build environment' section. Wrapper that provides constant Test Run metadata for all 'appMonRegisterTestRun' steps defined.
+
+* Required parameter:
+    * `systemProfile`
+* Optional parameters:
+    * `version*` - version descriptors aggregated to "major.minor.revision.milestone" pattern
+    * `marker` - string literal to provide a user-defined marker
+    * `recordSession` - boolean flag to enable session recording
+
+##### Examples of usage:
+Minimal set of parameters required:
+
+```groovy
+appMonBuildEnvironment(systemProfile: 'test_jenkins') {
+    //your pipeline logic
+}
+```
+All TestRun properties provided:
+```groovy
+appMonBuildEnvironment(marker: '1', recordSession: true, systemProfile: 'test_jenkins', versionMajor: '5', versionMilestone: '2', versionMinor: '4', versionRevision: '3') {
+    //your pipeline logic
+}
+```
+---
+#### **appMonRegisterTestRun**
+Has the same purpose as 'Dynatrace AppMon - Register Test Run' step - registers a Test Run used to differentiate subsets of tests. Uses Test Run's metadata defined in 'appMonBuildEnvironment' wrapper step.
+
+* Required parameter:
+    * `category` - category of TestRun(unit, uidriven, performance, webapi, external)
```

Comment:
space before '('
remove 'external' category, it's designed for a different purposes, and IMHO shouldn't be used this way